src/snmalloc/global/memcpy.h

```diff
@@ -156,7 +157,7 @@ namespace snmalloc
      */
     SNMALLOC_UNUSED_FUNCTION
     static constexpr size_t LargestRegisterSize =
-      std::max(sizeof(uint64_t), sizeof(void*));
+      proxy::max(sizeof(uint64_t), sizeof(void*));
```

Comment:
Could this use, then we wouldn't need to vendor std::max/std::min?
```suggestion
      bits::max(sizeof(uint64_t), sizeof(void*));
```

Reply:
done